technologies/app/cloudbeaver/cloudbeaver-1.2.0/image_test.yaml

```diff
@@ -1,7 +1,7 @@
 schemaVersion: "2.0.0"
 
 metadataTest:
-  env:
```

Comment:
Since the version v1.13.0 of gcr/container-structure-test, there are a change on naming:  Change env to envVars on metadataTest (https://github.com/GoogleContainerTools/container-structure-test/releases/tag/v1.13.0)